src/compiler/commandLineParser.ts

```diff
@@ -216,6 +216,7 @@ const libEntries: [string, string][] = [
     ["esnext.array", "lib.es2023.array.d.ts"],
     ["esnext.collection", "lib.es2023.collection.d.ts"],
     ["esnext.symbol", "lib.es2019.symbol.d.ts"],
+    ["esnext.arraybuffer", "lib.esnext.arraybuffer.d.ts"],

```

Comment:
@rbuckton Do you have an opinion abut we should add a separate lib entry for arraybuffer? This feels like the kind of thing that went in to earlier `es20xx.collection` entries.

Reply:
Resizable ArrayBuffers and growable SharedArrayBuffers are Stage 4 and will be in ES2024 when it is standardized, so a `lib.es2024.arraybuffer.d.ts`  might make sense at this point, though it would still only be referenced from `esnext` since we don't have a `--target es2024` yet..

Reply:
Or a `lib.es2024.collections.d.ts`, rather. I agree it makes more sense to try to keep the names somewhat consistent.